metriq/models/task.py

```diff
@@ -41,9 +41,9 @@ class Task(TeaClientModel):
         description (str): Task description.
     """
 
-    id: str
-    name: str
-    description: str
+    id: str("80")
```

Comment:
Mostly good, folks! However, we are not concerned with the data, at all, just the schema. In other words, all types in those models should default to empty values. Please remove the constructor arguments (with data) and check that your types work (for the schema).